### PR TITLE
Media Preview

### PR DESCRIPTION
**Pull request by aaosuarez**
6.UAP Project: Media Usability 
Created new "Preview" option in Media list to allow users to preview their files. Currently support .png, .jpg, .gif, .bmp, .mp3, .wav, .ogg, .mp4, and .webm for major browsers if they support them. Fallback messages for other file types and browsers that don't support the displaying the file types listed.
@halatmit @afmckinney @jisqyv

### CODE REVIEW

**Comment by josmas** [comment]
@a-suarez any chance you can push this and #763 to appspot for people to see it in action before reviewing?

**Comment by halatmit** [comment]
Thanks.  are you still planning to graduate? ;)




== Hal Abelson
hal@mit.edu
Prof. of Comp. Sci. and Eng.
MIT Dept. of Elec. Eng. and Comp. Sci.

File: appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/AssetList.java [hunk 1]
```diff
@@ -18,16 +19,12 @@
...
 import com.google.gwt.event.dom.client.ClickHandler;
 import com.google.gwt.event.logical.shared.SelectionEvent;
 import com.google.gwt.event.logical.shared.SelectionHandler;
-import com.google.gwt.user.client.ui.Composite;
-import com.google.gwt.user.client.ui.HTML;
-import com.google.gwt.user.client.ui.SimplePanel;
-import com.google.gwt.user.client.ui.Tree;
-import com.google.gwt.user.client.ui.TreeItem;
-import com.google.gwt.user.client.ui.VerticalPanel;
+import com.google.gwt.user.client.ui.*;
```
Comment (jisqyv) [inline appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/AssetList.java]: Please eliminate the * based import and replace it with explicit imports.

**Comment by jisqyv** [comment]
Looking good. I'll run some tests and if all is well I'll give it a thumbs up and merge (but I do need to test :-) ).

**Comment by afmckinney** [comment]
Tested but need to handle a few conner cases I'll fix and move to gerrit.

**Comment by ewpatton** [comment]
Note that because the branch no longer exists, I've manually merged this as fc2a232cc

**Pull request closed without merging**